packages/core/src/components/Tabs/Tabs.jsx

```diff
@@ -74,7 +94,24 @@ export class Tabs extends React.PureComponent {
     this.replaceState(href);
   }
 
-  // Filter children and return only TabPanel components
+  handleTabKeyPress(evt, panelId) {
```

Comment:
Nit: This handler gets called `onKeyDown`, not `onKeyPress`, so this method name isn't totally accurate.